salon_creator/lib/widgets/customButton.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
```

Comment:
In dart, the file names have to be snake cased, so that this file should be named custom_button.dart
The same applies for all such files in this PR